source/reduce/remove_unreferenced_instruction_reduction_opportunity_finder.cpp

```diff
@@ -52,6 +52,13 @@ RemoveUnreferencedInstructionReductionOpportunityFinder::
     result.push_back(MakeUnique<RemoveInstructionReductionOpportunity>(&inst));
   }
 
+  for (auto& inst : context->module()->debuginfo()) {
```

Comment:
It's a nice touch to add this to the fuzzer!

Reply:
:+1: